script.js

```diff
@@ -0,0 +1,116 @@
+const inputJs = document.querySelector(".input"),
+    toDoNewJs = inputJs.querySelector("#todo-new"),
+    listWaitingJs = document.querySelector(".waiting"),
+    listCompleteJs = document.querySelector(".complete");
+
+const TODOS = "toDos"; //key for saving todos in local storage
+
+let toDos = [];
+let maxId = 0;
+let countWaiting = 0, countComplete = 0;
```

Comment:
```suggestion
let waitingNum = 0, completeNum = 0;
```
 count가 함수 이름같은 느낌도 있어서, 변수라면 Num도 좋은 것 같아요 !

Reply:
맞는 말이세요!! 수정했습니다 ㅎㅎ

Reply:
개발스터디 후에 생각해보니
count가 앞에 있어서 그렇게 느껴졌던것 같아요 ^^
waitingCount 는 어떨까요 ㅎㅎ